modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -53,36 +53,43 @@ def parse_table_row(row)
     end
 
     def parse_form_row(line, url)
-      title = line.css('font').text
       form_name = line.css('a').first.text
       form = VaForms::Form.find_or_initialize_by form_name: form_name
       current_sha256 = form.sha256
-      form.title = title
-      issued_string = line.css('td:nth-child(3)').text
+      form.title = line.css('font').text
       revision_string = line.css('td:nth-child(4)').text
-      form.first_issued_on = parse_date(issued_string) if issued_string.present?
       form.last_revision_on = parse_date(line.css('td:nth-child(4)').text) if revision_string.present?
       form.pages = line.css('td:nth-child(5)').text
-      form_url = url.starts_with?('http') ? url : get_full_url(url)
+      form_url = url.starts_with?('http') ? url.gsub('http:', 'https:') : get_full_url(url)
```

Comment:
@bastosmichael  Are we sure all urls are https? Or they will be https?

Reply:
Yes I was able to confirm that all 532 forms do support https and in fact the reason some of the forms were failingn previously is because not converting from http to https was actually not getting redirected properly from http to https. Don't know if that changes in the future but I doubt any new forms will be added without https support and redirection at least.